src/effects/Distortion.cpp

```diff
@@ -66,8 +66,7 @@ const EffectParameterMethods& EffectDistortion::Parameters() const
    > parameters{
       [](EffectDistortion &e, EffectSettings &settings, EffectDistortionSettings &p,
          bool updating) {
-         if (!updating)
-            e.mMaster.mThreshold = DB_TO_LINEAR(p.mThreshold_dB);
+
```

Comment:
Now that the lambda's body shrinks to nothing, just remove it.  `parameters` needs no constructor arguments at all.

Reply:
ok, done